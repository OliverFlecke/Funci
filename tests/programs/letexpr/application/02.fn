main :: Int = 
    let g x :: (Int -> Int) = x * x
    in let f :: (Int -> Int) = g
        in g 7
