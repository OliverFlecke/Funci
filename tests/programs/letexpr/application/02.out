Number 49 (Unit [])